easybuild/easyconfigs/n/nvtop/nvtop-1.2.2-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'nvtop'
+version = '1.2.2'
+
+homepage = 'https://github.com/Syllo/nvtop'
+description = 'htop-like GPU usage monitor'
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['d6ca7f467cc1048ebd1ebf7add753650c5d7c8868752c0c57ed2c85df62f5810']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+]
+
+# Note: software dlopen's libraries from the OS installed drivers, thus no specific CUDA dependency.
```

Comment:
@crubb @Micket  would it make sense then to remove the CUDA dependency from the existing `nvtop-1.2.1-GCCcore-10.2.0.eb` easyconfig, for consistency with `nvtop-1.2.1-GCCcore-10.3.0.eb` and these new ones?

Reply:
Happy to oblige (see new commit), unless there are objections from @Micket? Glad to oblige (see new commit), unless there are objections from @Micket?

Reply:
no objections